client/transport/src/ws/mod.rs

```diff
@@ -614,7 +614,15 @@ impl TryFrom<url::Url> for Target {
 #[cfg(feature = "tls")]
 fn build_tls_config(cert_store: &CertificateStore) -> Result<tokio_rustls::TlsConnector, WsHandshakeError> {
 	let config = match cert_store {
+		#[cfg(feature = "tls-rustls-platform-verifier")]
 		CertificateStore::Native => rustls_platform_verifier::tls_config(),
+		#[cfg(not(feature = "tls-rustls-platform-verifier"))]
+		CertificateStore::Native => {
```

Comment:
I don't think it's necessary to add such complicated feature-gated stuff for such a niche feature.

Not super elegant to throw an error here with "the wrong configuration" when it could be a compile error but I'm okay with this.

Reply:
are you okay with my changes @oleonardolima?

Reply:
As we removed the `Default` implementation now I think it's fine.